lang/LangPrimSource/PyrUnixPrim.cpp

```diff
@@ -305,18 +305,31 @@ int prUnix_Errno(struct VMGlobals *g, int numArgsPushed)
 
 #include <time.h>
 
-static void fillSlotsFromTime(PyrSlot * result, struct tm* tm, std::chrono::system_clock::time_point const & now)
+// Set only Date members related to the tm struct: YMD, HMS and
+// weekday (not rawSeconds)
+static void fillSlotsFromTimeStruct(PyrSlot * result, struct tm* tm)
 {
 	PyrSlot *slots = slotRawObject(result)->slots;
 
 	SetInt(slots+0, tm->tm_year + 1900);
-	SetInt(slots+1, tm->tm_mon + 1); // 0 based month ??
+	SetInt(slots+1, tm->tm_mon + 1); // 0 based month
 	SetInt(slots+2, tm->tm_mday);
 	SetInt(slots+3, tm->tm_hour);
 	SetInt(slots+4, tm->tm_min);
 	SetInt(slots+5, tm->tm_sec);
 	SetInt(slots+6, tm->tm_wday);
-	SetFloat(slots+7, std::chrono::duration_cast<std::chrono::nanoseconds>(now.time_since_epoch()).count() * 1.0e-9);
+}
+
+// Set all Date members (including rawSeconds)
```

Comment:
just a comment: in sclang, "member variables" are usually called "instance variables"

Reply:
Okay, thanks, updating all uses of "member" to "instance variables".